### PR TITLE
Add shellcheck

### PR DESCRIPTION
**Pull request by pascal-airy**

### CODE REVIEW

File: lint/shellcheck.bzl [hunk 1]
```diff
@@ -0,0 +1,20 @@
+def shellcheck(
+        name = "shellcheck",
+        scripts = None,
+        rootpath = None):
+    scripts = scripts if scripts else []
```
Comment (chrismatix) [inline lint/shellcheck.bzl:5]: Defaulting to a `glob` seems more useful :)

File: lint/shellcheck.bzl [hunk 2]
```diff
@@ -0,0 +1,20 @@
+def shellcheck(
+        name = "shellcheck",
+        scripts = None,
+        rootpath = None):
```
Comment (chrismatix) [inline lint/shellcheck.bzl:4]: Remove

File: lint/shellcheck.bzl [hunk 3]
```diff
@@ -0,0 +1,21 @@
+def shellcheck(
+        name = "shellcheck",
+        scripts = None,
```
[the remark this answers is not in the document]
Comment (pascal-airy) [reply lint/shellcheck.bzl:3]: but isn't that confusing with the real `srcs` argument of the sh_binary rule?

**Review by chrismatix — COMMENTED** [review]
Please add docs to `README.md` then this is ready to go

File: lint/shellcheck.bzl (hunk 3 above)
[the remark this answers is not in the document]
Comment (chrismatix) [reply lint/shellcheck.bzl]: To the user of the macro, the sh implementation is not immediately obvious. So they can't really get confused there. However, it is common to expect a `srcs` attribute to describe a set of source file inputs.

**Review by chrismatix — APPROVED** [review]

**Pull request merged**